test/tests/functional/pbs_reservations.py

```diff
@@ -203,7 +203,11 @@ def test_sched_cycle_starts_on_resv_end(self):
         self.scheduler.log_match(
             jid + ";" + msg,
             max_attempts=30)
+        
+        a = {'reserve_state': (MATCH_RE, 'RESV_RUNNING|2')}
+        self.server.expect(RESV, a, rid)
+        
         self.server.log_match(
             rid.split('.')[0] + ";deleted at request of pbs_server",
-            id=rid.split('.')[0], max_attempts=30)
+            id=rid.split('.')[0], interval=5)
```

Comment:
You do the rid.split() twice in this log match.  Do it outside and save the value into a variable so you only need to do it once.